docs/cl/libvirt.md

```diff
@@ -0,0 +1,245 @@
+# Libvirt
+
+In this tutorial, boot and provision a Kubernetes v1.11.0 using a local libvirt instance.
+
+The libvirt archirecture is meant for development and testing, rather than long-term use. It does not support load-balancing between multiple API servers, for example.
```

Comment:
Phrasing is crucial here. Development/testing can't be grounds for the architecture or conventions to differ between Typhoon's platforms. We can say its only suitable for local user though, that's fine.

All Typhoon clusters should allow multi-master setups. This can be treated just like bare-metal, where load balancing is left to how the end user chooses to resolve / balance across masters.

Reply:
Updated the phrasing